### ISSUE TITLE
0.7.0 release on RubyGems differs from GitHub release

### ISSUE THREAD

**Issue by CpuID** [xeger/docker-compose#24]
Looks like the latest git commits somehow didn't make it to Rubygems, see below:

GitHub release tarball:

```
ruby-2.2.0 [nathan@NS-MB-Pro docker-compose-0.7.0]$ wget -q https://github.com/xeger/docker-compose/archive/v0.7.0.tar.gz
ruby-2.2.0 [nathan@NS-MB-Pro docker-compose-0.7.0]$ tar xzf v0.7.0.tar.gz 
ruby-2.2.0 [nathan@NS-MB-Pro docker-compose-0.7.0]$ grep "def rm" docker-compose-0.7.0/lib/docker/compose/session.rb 
    def rm(*services, force:false, volumes:false)
```

RubyGems download .gem:

```
ruby-2.2.0 [nathan@NS-MB-Pro docker-compose-0.7.0]$ wget -q https://rubygems.org/downloads/docker-compose-0.7.0.gem
ruby-2.2.0 [nathan@NS-MB-Pro docker-compose-0.7.0]$ tar xzf docker-compose-0.7.0.gem
ruby-2.2.0 [nathan@NS-MB-Pro docker-compose-0.7.0]$ tar xzf data.tar.gz
ruby-2.2.0 [nathan@NS-MB-Pro docker-compose-0.7.0]$ grep "def rm" lib/docker/compose/session.rb 
    def rm(*services, force:false, volumes:false, all:true)
```

New release time maybe, 0.7.1?

**Comment by xeger** [comment]
D'oh! Apologies. I'll look into your open PR and then publish 0.7.1

**Comment by xeger** [comment]
Fixed by publishing 0.8.0, containing new features + lingering fixes that were missing from 0.7.1
